### ISSUE TITLE
Admin UI way to trigger a VersionPress pull

### ISSUE THREAD

**Issue by candrews** [versionpress/versionpress#1357]
There should be a way to trigger `wp vp pull` from the web interface at `/wp-admin/admin.php?page=versionpress` Currently, one has to SSH to the system and run `wp vp pull` from the shell.

**Comment by borekb** [comment]
I agree. Generally, workflow / sync features are only covered by WP-CLI commands right now; it would indeed be nice to have a UI for them in the future.
